From: Agustin Henze <tin@sluc.org.ar>
Date: Mon, 18 Nov 2013 16:28:05 -0300
Subject: Fix documentation,
 intersphinx mapping * Removed URL of the flask and django-assets
 docs and changed path of the python doc in the intersphinx mapping.

---
 docs/conf.py | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/docs/conf.py b/docs/conf.py
index 54dc0d7..1180b40 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -201,9 +201,7 @@ latex_documents = [
 #latex_use_modindex = True
 
 intersphinx_mapping = {
-    'python': ('http://docs.python.org/', None),
-    'flask': ('http://elsdoerfer.name/docs/flask-assets/', None),
-    'django': ('http://elsdoerfer.name/docs/django-assets/', None),
+    'python': ('/usr/share/doc/python2.7/html', None),
 }
 
 extlinks = {'django': ('http://elsdoerfer.name/docs/django-assets/%s.html', None)}
